s.to/dist/metadata.json

```diff
@@ -9,20 +9,12 @@
         "de": "Streame mehr als 3500 Episoden von deiner lieblings serie kostenlos!",
         "nl": "Stream meer dan 3500 afleveringen van je favorite serie gratis!" 
     },
-    "url": [
-        "s.to",
-        "board.s.to",
-        "serienstream.to",
-        "serienstream.sx"       
-    ],
-    "version": "1.2",
+    "url": [s.to","board.s.to","serienstream.sx"],
```

Comment:
You're missing a **"** on the url array.

Reply:
Done! :D